news/2018-07-20-the-qat-gazette-june-2018.md

```diff
@@ -0,0 +1,98 @@
+
+Community members were able to share some of their opinions on a plethora of topics, including changes to the Beatmap Nominators subdivision system, beatmap veto mediation, tighter quality standards and review of our Beatmap Nominators, introduction of QAT interviews, and changes to Beatmap Nominator applications.
+
+Don't worry if you weren't able to make it for the meeting – you can still catch up and get the lowdown using the Beatmap Management Forum [summary article](https://osu.ppy.sh/community/forums/topics/762133).
+
+Given the meeting's success, we naturally plan to host more public discussions in the future, so stay tuned as we’ll make sure to announce them in advance right here in the Gazette before anywhere else. These are hosted in the [public osu!dev Discord server](https://discord.gg/ppy), so be sure to join and verify yourself if you haven’t done so already.
+
+### osu!mania Ranking Criteria Draft
+
+After three long months, the new osu!mania mode-specific Ranking Criteria draft is out for review now. We've taken community feedback and incorporated it into the current Ranking Criteria draft, including clarifications and additions that are hopefully less vague and more intuitive to read.
+
+We've also added a few new Rules and converted some old Rules into Guidelines, to be more flexible and afford more freedom to the hard working mappers and modders who bring new content into the Ranked section.
+
+Stop by and give the [current draft](https://gist.github.com/Okorin/1a5f5d6f489494c26b8a4fec13ba69ba) a read, and drop your feedback and opinions in the [discussion thread](https://osu.ppy.sh/community/forums/topics/719936) to help shape the future and direction of osu!mania mapping!
+
+### New Quality Assurance Team additions
+
+This month, we welcome a new piano masher to the Quality Assurance Team. Formerly a long standing member of the Beatmap Nominators, [Maxus](https://osu.ppy.sh/users/4335785) will be assisting the Quality Assurance Helpers with checking and disqualifying beatmaps, and giving [Feerum](https://osu.ppy.sh/users/4815717) and myself a helping hand on the osu!mania community side of things. We're glad to have him on board!
+
+On the osu! side of the coin, another new team member has arisen: [Hobbes2](https://osu.ppy.sh/users/8157492). Hobbes' recent output of checked beatmaps in the Quality Assurance Helpers has been frankly astonishing, and we hope that trend continues in the QAT to keep on top of the ever-growing pool of osu! beatmaps making their way to the Qualified section.
+
+### Quality Assurance Team retirements
+While not a retirement exactly, the Quality Assurance Team underwent a bit of spring cleaning this month, where we evaluated the direction and focus we want to have as a team, specifically one that is geared mostly towards disqualifications and beatmap checking.
```

Comment:
Add a line between the header and paragraph.

Reply:
✅